edge/pkg/devicetwin/dtcontroller.go

```
@@ -123,7 +123,10 @@ func (dtc *DTController) distributeMsg(m interface{}) error {
 	if message.Msg.GetParentID() != "" {
 		log.LOGGER.Infof("Send msg to the %s module in twin", dtcommon.CommModule)
 		confirmMsg := dttype.DTMessage{Msg: model.NewMessage(message.Msg.GetParentID()), Action: dtcommon.Confirm}
-		dtc.DTContexts.CommTo(dtcommon.CommModule, &confirmMsg)
+		err := dtc.DTContexts.CommTo(dtcommon.CommModule, &confirmMsg)
+		if err != nil {
+			return errors.New("can not communicate with module %v, error message is: %v", dtcommon.CommModule, err)
```

Do not wrap errors, please return the error directly.